src/lwpUserAgent.js

```diff
@@ -217,14 +217,20 @@ export default class extends lwpRenderer {
     this._emit("redial.update", this, this._redialTarget);
   }
 
-  call(target = null, call_options = {}) {
+  call(target = null, custom_headers = [], call_options_anonymous = false) {
     let options = {
-      ...call_options,
       data: { lwpStreamId: lwpUtils.uuid() },
-      extraHeaders: [...(call_options.extraHeaders || []), ...this._config.custom_headers.establish_call],
-      anonymous: call_options.anonymous || false
+      extraHeaders: [...custom_headers, ...this._config.custom_headers.establish_call],
     };
-
+    if (typeof call_options_anonymous === 'boolean') {
+      options.anonymous = call_options_anonymous;
+    }
+    if (typeof call_options_anonymous === 'object') {
+      options.rtcOfferConstraints = {
+        offerToReceiveVideo:  call_options_anonymous.enable_video || false
```

Comment:
I would prefer a more descriptive parameter name rather than "enable_video"... technically it isn't enabling video it is simply informing the server that it wants to RECEIVE video if available... how about "receive_video" or "unsymmetrical_video" .... or "demand_video" or "petition_for_video" or "hit_up_for_video" or "whanna_neflix_and_chill" ;)

Reply:
done renamed as just `options` 👍 😄